news/4992.feature.rst

```diff
@@ -0,0 +1 @@
+Internal to pipenv, the utils.py was split into a utils module with unused code removed.
```

Comment:
Note: that in the fast-follow linting PR I added a new type `process` same as Pip has, and this news fragment is converted to a `process` type in the subsequent PR.  I went back and forth on feature vs process for this one, but its not an end-user facing feature.